app.py

```diff
@@ -299,23 +303,47 @@ def create_message():
 
 @app.route('/message', methods=['GET'])
 def get_message():
-    club_id = request.args.get('club_id')
-    Session, engine = dbconnect(db_options)
-    session = Session()
-    messages = session.query(Message).filter(Message.club_id == club_id).all()
-    #print (messages)
-    ret_messages = []
-    for m in messages:
-        ret_messages.append(
-            {
-                'id': m.id,
-                'club_id': m.club_id,
-                'message': m.message
-            }
-        )
-    response = Response(json.dumps(ret_messages))
-    response.headers['Access-Control-Allow-Origin'] = '*'
-    return response
+    if (request.args.get('club_id')):
+        club_id = request.args.get('club_id')
+        Session, engine = dbconnect(db_options)
+        session = Session()
+        messages = session.query(Message).filter(Message.club_id == club_id).all()
+        #print (messages)
+        ret_messages = []
+        for m in messages:
+            ret_messages.append(
+                {
+                    'id': m.id,
+                    'club_id': m.club_id,
+                    'message': m.message
+                }
+            )
+        response = Response(json.dumps(ret_messages))
+        response.headers['Access-Control-Allow-Origin'] = '*'
+        return response
+    elif (request.args.get('user_id')):
+        Session, engine = dbconnect(db_options)
+        session = Session()
+        #print (messages)
+        user_id = request.args.get('user_id')
+        print (user_id)
+        ret_messages = []
+        allClubs = userClubs(session,user_id)
+        print (allClubs)
+        for c in allClubs:
+            messages = session.query(Message).filter(Message.club_id == c.id).all()
+            for m in messages:
```

Comment:
this part looks a whole lot like the part above.... consider making a function that accepts a club_id and returns all the messages for the club